apps/validate/validate.py

```diff
@@ -16,11 +16,35 @@
 logger = logging.getLogger(__name__)
 
 
+def check_json(doc, field, value):
+    if isinstance(doc, dict):
+        if field in doc and doc[field] == value:
+            return True
+        for key in doc:
+            if check_json(doc[key], field, value):
+                return True
+        return False
+    elif isinstance(doc, list):
+        for item in doc:
+            if check_json(item, field, value):
+                return True
+        return False
+    else:
+        return False
+
+
 class SchemaValidator(cerberus.Validator):
     def _validate_type_picture(self, field, value):
         """Allow type picture in schema."""
         pass
 
+    def _validate_mandatory_in_list(self, mandatory, field, value):
```

Comment:
this won't be triggered when using `validator.validate` right? so how it should be used?